src/interfaces/IAccountLoupe.sol

```diff
@@ -12,17 +12,16 @@ struct ExecutionHook {
 }
 
 interface IAccountLoupe {
-    /// @notice Config for an execution function, given a selector.
-    struct ExecutionFunctionConfig {
-        address plugin;
-        FunctionReference validationFunction;
-    }
-
-    /// @notice Get the validation functions and plugin address for a selector.
+    /// @notice Get the plugin address for a selector.
     /// @dev If the selector is a native function, the plugin address will be the address of the account.
     /// @param selector The selector to get the configuration for.
-    /// @return The configuration for this selector.
-    function getExecutionFunctionConfig(bytes4 selector) external view returns (ExecutionFunctionConfig memory);
+    /// @return plugin The plugin address for this selector.
+    function getExecutionFunctionHandler(bytes4 selector) external view returns (address plugin);
```

Comment:
I wonder if it's worth supporting installations of different plugins that share the same execution function selector. So this could return `address[]`, and `getValidationFunctions` below could take in an additional arg `address`.

Reply:
I think the reason we can't really do that is because the account has to know where to route calls coming into the `fallback` function, and we don't necessarily have that context. We could try to push something where the handler gets specified at the end of calldata, but it seems like a lot of complexity for not that much upside.

We may want to change the loupe interfaces though, but that's probably something we should look at after the majority of v0.8 refactors are complete.

Reply:
If we're no longer installing execution functions, what are we handling in the fallback function?

Reply:
We are still installing execution functions if a plugin defines it, we've just decoupled installing validation functions and installing execution functions.

Reply:
Ah, right. Off topic now, but do you think it'd make sense / be feasible to just remove that entirely? I.e., enforce routing all plugin execution functions via `execute` / `executeBatch`?

Reply:
I guess it'd become difficult to support adding new functions to accounts to be compatible with new ERCs down the road (e.g., token callback handlers).

Reply:
I think the intention behind adding execution functions would be to make the account itself conform to a new interface - i.e. [ERC-173](https://eips.ethereum.org/EIPS/eip-173). Conforming to these interfaces requires that the functions aren't wrapped in something like `execute`/`executeBatch`, so we should still support execution functions in some capacity.